engines/director/lingo/lingo-builtins.cpp

```diff
@@ -1823,9 +1823,19 @@ void LB::b_numberoflines(int nargs) {
 void LB::b_numberofwords(int nargs) {
 	Datum d = g_lingo->pop();
 
-	warning("STUB: b_numberofwords");
-	d.toInt();
-	d.u.i = 0;
+	d.toString();
+	int numberofwords = 0;
+	Common::String contents = *d.u.s;
+	for (uint32 i=1; i < d.u.s->size(); i++) {
+		if (Common::isSpace(contents[i]) and not Common::isSpace(contents[i - 1]))
```

Comment:
I am not sure that `not` keyword is compatible enough. Please use `!`

Reply:
Same goes for the `and` keyword, please use &&